### DISCUSSION TITLE
POS Tags full name

### DISCUSSION THREAD

**Discussion by laxmimerit** [explosion/spaCy#9345] [Help: Coding & Implementations] [answered]
Hi,
Is there a way to change the POS to the actual description?
For example, instead of RB - we will write "adverb" and this will be plotted in the bar chart?

**Comment by polm** [comment, marked as answer]
What do you mean by "bar chart"?

You can get an explanation of most abbreviations used in spaCy by using `spacy.explain("NNP")` or whatever.
